Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/README.md

```diff
@@ -0,0 +1,4728 @@
+                        "id": "dummy_id",
+                        "inherited_from": "",
+                        "name": "Endpoint standard sensor settings at policy scope",
+                        "parameters": {
+                            "inline_blocking_mode": "disabled"
+                        }
+                    },
+                    {
+                        "category": "data_collection",
+                        "description": "Collects events created when a process loads a common library. Enabling this will increase the number of events reported for expected process behavior.",
+                        "id": "dummy_id",
+                        "inherited_from": "psc:region",
+                        "name": "Prevalent Module Exclusions",
+                        "parameters": {
+                            "enable_prevalent_module_event_collection": false
+                        }
+                    }
+                ],
+                "rules": [],
+                "sensor_configs": [
+                    {
+                        "category": "sensor_settings",
+                        "description": "Manages sensor settings specific to endpoint standard product",
+                        "id": "dummy_id",
+                        "inherited_from": "",
+                        "name": "Endpoint standard sensor settings at policy scope",
+                        "parameters": {
+                            "inline_blocking_mode": "disabled"
+                        }
+                    }
+                ],
+                "sensor_settings": [
+                    {
+                        "name": "ALLOW_UNINSTALL",
+                        "value": "true"
+                    },
+                    {
+                        "name": "LEARNING_MODE",
+                        "value": "0"
+                    },
+                    {
+                        "name": "ALLOW_INLINE_BLOCKING",
+                        "value": "true"
+                    }
+                ],
+                "update_time": 1720539000191
+            },
+            "priorityLevel": "MEDIUM"
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Policy created successfully
+>|Id|Description|Name|Priority Level|Is System|
+>|---|---|---|---|---|
+>| 170044 | aaaaaaaa | test4 | MEDIUM | false |
+
+
+### cbd-update-policy
+
+***
+Update an existing policy on the CB Defense backend.
+
+#### Base Command
+
+`cbd-update-policy`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| description | A description of the policy. Can be multiple lines. | Optional | 
+| name | A unique one-line name for the policy. | Optional | 
+| priorityLevel | The priority score associated with sensors assigned to this policy. Possible values: "MISSION_CRITICAL", "HIGH", "MEDIUM", and "LOW". Possible values are: MISSION_CRITICAL, HIGH, MEDIUM, LOW. | Optional | 
```

Comment:
Delete the extra Possible values: "MISSION_CRITICAL", "HIGH", "MEDIUM", and "LOW".